domains/aysko.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "aysko-discord",
+           "email": "marckelyfc@icloud.com",
+           "discord": "370342658033319936"
+        },
+    
+        "record": {
+            "CNAME": "relaxed-sable-26f2ce.netlify.app."
```

Site not found, also technically CNAME is invalid
```suggestion
            "CNAME": "relaxed-sable-26f2ce.netlify.app"
```